trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_RHS_matrix.py

```diff
@@ -99,15 +126,155 @@ def compute_RHS_matrix(delta,phi,conditions,geometry,propeller_wake_model,bemt_w
+    
+    #unpack/rename variables from VD
+    X      = VD.XCH
+    YY     = VD.YCH
+    ZZ     = VD.ZCH 
+    XBAR   = VD.XBAR
+    ZBAR   = VD.ZBAR
+
+    CHORD  = VD.chord_lengths[0,:]
+    DELTAX = 0.5/RNMAX 
+    
+    # LOCATE VORTEX LATTICE CONTROL POINT WITH RESPECT TO THE
+    # ROTATION CENTER (XBAR, 0, ZBAR). THE RELATIVE COORDINATES
+    # ARE XGIRO, YGIRO, AND ZGIRO.
+    XGIRO = X + CHORD*DELTAX - np.repeat(XBAR, RNMAX[LE_ind])
+    YGIRO = YY 
+    ZGIRO = ZZ - np.repeat(ZBAR, RNMAX[LE_ind])  
+    
+    # VX, VY, VZ ARE THE FLOW ONSET VELOCITY COMPONENTS AT THE LEADING
+    # EDGE (STRIP MIDPOINT). VX, VY, VZ AND THE ROTATION RATES ARE
+    # REFERENCED TO THE FREE STREAM VELOCITY.    
+    VX = (COSCOS - PITCH*ZGIRO + YAW  *YGIRO)
+    VY = (COSIN  - YAW  *XGIRO + ROLL *ZGIRO)
+    VZ = (SINALF - ROLL *YGIRO + PITCH*XGIRO)
+
+    # CCNTL AND SCNTL ARE DIRECTION COSINE PARAMETERS OF TANGENT TO
+    # CAMBERLINE AT LEADING EDGE.
+    # SLE is slope at leading edge only     
+    SLE    = np.repeat(VD.SLE, RNMAX[LE_ind])
+    CCNTL  = 1. / np.sqrt(1.0 + SLE**2) 
+    SCNTL  = SLE *CCNTL
+    phi_LE = np.repeat(phi[:,LE_ind]  , RNMAX[LE_ind], axis=1) 
+    COD    = np.cos(phi_LE)
+    SID    = np.sin(phi_LE)
+
+    # COMPUTE ONSET FLOW COMPONENT ALONG THE OUTWARD NORMAL TO
+    # THE SURFACE AT THE CONTROL POINT, ALOC.
+    ALOC  = VX *SCNTL + VY *CCNTL *SID - VZ *CCNTL *COD    
+    
+    # COMPUTE VELOCITY COMPONENT ALONG X-AXIS INDUCED BY THE RIGID
+    # BODY ROTATION, ONSET.    
+    ONSET = - PITCH *ZGIRO + YAW *YGIRO
+       
+    # Body-Frame RHS calculation----------------------------------------------------------
+    # Add wake and rotation effects to the freestream
+    Vx_rotation       = -PITCHQ*ZGIRO + YAWQ  *YGIRO
+    Vy_rotation       = -YAWQ  *XGIRO + ROLLQ *ZGIRO
+    Vz_rotation       = -ROLLQ *YGIRO + PITCHQ*XGIRO
+    
+    Vx                = V_distribution*np.cos(aoa_distribution)*np.cos(PSI_distribution) + Vx_rotation - Vx_ind_total
+    Vy                = V_distribution*np.cos(aoa_distribution)*np.sin(PSI_distribution) + Vy_rotation + Vy_ind_total
+    Vz                = V_distribution*np.sin(aoa_distribution)                          + Vz_rotation - Vz_ind_total
+    V_distribution    = np.sqrt(Vx**2 + Vy**2 + Vz**2 )
+    
+    aoa_distribution  = np.arctan(Vz/ np.sqrt(Vx**2 + Vy**2) )    
+    PSI_distribution  = np.arctan(Vy / Vx)    
+    
+    # compute RHS: dot(v, panel_normals)
+    # note there is a subtle difference bewtween the normals used here and VD.normals
+    panel_normals    = np.array([np.sin(delta), np.cos(delta)*np.sin(phi), -np.cos(delta)*np.cos(phi)]).T
+    V_unit_vector    = (np.array([Vx,Vy,Vz])/V_distribution).T
+    RHS_from_normals = np.sum(-V_unit_vector*panel_normals, axis=2).T
+    
+    #pack values--------------------------------------------------------------------------
+    use_VORLAX_RHS = getattr(settings, 'use_VORLAX_matrix_calculation', False)
```

Comment:
just use normal syntax. This getattr syntax is going to confuse new coders

Reply:
Got it. Need a sec to test if this breaks any regressions (e.g. they might not define settings.use_VORLAX_matrix_calculation, thereby throwing a getattribute error)

Reply:
Done